includes/class-edd-cache-helper.php

```diff
@@ -32,21 +32,19 @@ public function init() {
 
 		if ( false === ( $page_uris = get_transient( 'edd_cache_excluded_uris' ) ) ) {
 
-			global $edd_options;
-
-			if( empty( $edd_options['purchase_page'] ) || empty( $edd_options['success_page'] ) ) {
+			if( ! edd_get_option( 'purchase_page', false ) || ! edd_get_option( 'success_page' ) ) {
```

Comment:
No. The original didn't have a default, and we can't predict what pages will or will not exist with any certainty. If we assume a page exists and it doesn't (or isn't published) things don't work right.